composer.json

```diff
@@ -14,8 +14,9 @@
            }
        },
     "require": {
-        "php": "^7.4",
-        "goaop/framework": "^3.0",
+        "php": "^8.2",
+        "goaop/framework": "^4.0@dev",
+        "goaop/parser-reflection": "^4.0@dev",
```

Comment:
@marcovtwout I can merge this PR, but I think that it would be better to delay release until goaop/framework 4.0.0 is released.

I added branch alias below.

Reply:
Created PR: https://github.com/Codeception/AspectMock/pull/212